packages/medusa/src/api/routes/admin/customer-groups/get-customer-group.ts

```diff
@@ -0,0 +1,49 @@
+import { CustomerGroupService } from "../../../../services"
+import { FindParams } from "../../../../types/common"
+import { validator } from "../../../../utils/validator"
+import { defaultAdminCustomerGroupsRelations } from "."
+
+/**
+ * @oas [get] /customer-group/{id}
+ * operationId: "GetCustomerGroups"
```

Comment:
`GetCustomerGroups` is the opertionId for the list endpoint :)

```suggestion
 * operationId: "GetCustomerGroupsGroup"
```